physionet-django/console/templates/console/training_process.html

```diff
@@ -0,0 +1,46 @@
+{% extends "console/base_console.html" %}
+
+{% load static %}
+
+{% block title %}To be changed{% endblock %}
+
+{% block content %}
+<h1>To be changed</h1>
+<hr>
+
+<div class="card-columns row">
+    <div class="col-sm-5">
+        <div class="card mb-4">
+            <div class="card-header">
+                Training information
+            </div>
+            <div class="card-body">
+                {% if training.completion_report %}
+                    Document: <a href="{% url 'training_report' training.pk %}" target="_blank">Training report</a><br>
+                {% endif %}
+                {% if training.completion_report_url %}
+                    URL: <a href="{{ training.completion_report_url }}" target="_blank">{{ training.completion_report_url }}</a><br>
+                {% endif %}
+                Submitted on: {{ training.application_datetime|date }}
+            </div>
+        </div>
```

Comment:
To help the reviewer, I think we probably need to display an "Application information" card on this page too (similar to the one in the credentialing section). The reviewer will want to be able to compare the details of the applicant with the information on the form (e.g. name, email address, etc).